src/main/realms.c

```diff
@@ -2943,7 +2943,7 @@ int home_server_afrom_file(char const *filename)
 		return -1;
 	}
 
-	cs = cf_section_alloc(NULL, "home", filename);
+	cs = cf_section_alloc(NULL, "home_server", filename);
```

Comment:
I don't think this is correct.  The call to `cf_file_read()` will read the file as a *subsection* of `cs` here.  So the name of `cs` doesn't matter.  It could be anything.

Reply:
yeah! I updated it. done